tests/sentry/tasks/test_post_process.py

```diff
@@ -54,8 +56,7 @@ def __eq__(self, other):
         return matching_id
 
 
-@region_silo_test
-class PostProcessGroupTest(TestCase):
+class CorePostProcessGroupTestMixin:
```

Comment:
Nit: would it make sense to have these mixings inherit from `TestCase` and introduce an abstract class:
```

class PostProcessGroupBaseMixin:
    @abstractmethod
    def create_event(self, data, project_id):
        raise NotImplementedError
```

That the appropriate Mixings will inherit from as well as the real test class.

Reply:
The problem is that if you inherit from `TestCase` then pytest picks up the tests in the mixins and tries to run them. 

tbh I don't love this mixin pattern a lot, but it seems like the easiest way to factor these in the short term.

Reply:
Yeah, not a huge fan of the unresolved references but I see what you mean. I think its fine for now until we can clean up these tests.

Reply:
Realised that we can inherit from `BaseTestCase` and not have these issues, so used that as the base for mixins